### DISCUSSION TITLE
Unable to see incoming message log in postal web console

### DISCUSSION THREAD

**Discussion by DeepsDevOps** [postalserver/postal#3065] [Help with using Postal]
I am using postal 3.3.4. I am able to see outgoing messages status but not able to see only sent messages in incoming messages.

In the database all entries are available but not able to see in the web console.

**Comment by willpower232** [comment]
Incoming messages are those which have been sent to domains in your Postal by other mail servers on the internet, I'm not sure what you are looking for?

**Reply by DeepsDevOps** [reply]
Thanks for reply. Herewith i am attaching screenshot. before upgrade  i was able to get details but after upgraded from 3.3.3 to 3.3.4 , I am not get sent messages status for incoming emails. 
![postal-incoming-messages](https://github.com/user-attachments/assets/61a0fa41-a036-445e-ab64-b92db758e4c0)

**Reply by willpower232** [reply]
when you say you do not get the sent status, what do you see instead?

**Reply by DeepsDevOps** [reply]
Hello @willpower232 ,

The exact issue that I am facing is that when a user from any other mail server like gmail.com or yahoo.com etc sends the mail to my postal mail server domain then we receive the mail in our inbox but the message logs then we can see in the incoming section of messages is not showing to us from 3rd of July.

![P3](https://github.com/user-attachments/assets/13f5e846-cbf1-43c8-b8a9-a6259ef8b408)
![P2](https://github.com/user-attachments/assets/2d4e6733-bbb1-47cc-a205-a721ef91fd10)
Screenshots of both the scenario one where the logs are show and the second where the logs are not shown. I can see only hardfail logs and processed not sent.

I checked the database for the logs and the logs are been written in the database. Its just that it is not been shown in the Web Console UI.

So I wanted to know your inputs from you end and can you help us fix this.

Current Postal version: V3.3.4




Thanks,

**Reply by willpower232** [reply]
What you have there are two different kinds of incoming message, the INVOICE ones are direct emails to your Postal server, the return path ones are bounces of a few varieties. If you click into the return path ones, does it give you more detail on the right hand side?

**Reply by DeepsDevOps** [reply]
Before 3 July  i was able to see incoming messages log in web UI as you can see INVOICE one & the status of the same is **sent** but currently I can see only **hardfail** and **processed** messages logs. no any incoming messages display in web UI with status **sent**.
